docs/db-migration/prototypes/audit-log.html

```
@@ -0,0 +1,303 @@
+    <main class="max-w-7xl mx-auto px-6 py-6">
+        <!-- Page Header -->
+        <div class="mb-6">
+            <h2 class="text-2xl font-semibold tracking-tight">Audit Log</h2>
+            <p class="text-sm text-muted mt-1">Track all configuration changes across components.</p>
+        </div>
+
+        <!-- Filters -->
+        <div class="flex items-center gap-3 mb-4">
+            <div class="relative flex-1 max-w-xs">
+                <svg class="absolute left-3 top-1/2 -translate-y-1/2 w-4 h-4 text-muted" fill="none" stroke="currentColor" viewBox="0 0 24 24"><path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"/></svg>
+                <input type="text" placeholder="Filter by user..." class="field-input pl-9">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)